BUILD.bazel

```
@@ -87,6 +87,7 @@ alias(
 # $ bazel run //:python-requirements.update
 compile_pip_requirements(
     name = "python-requirements",
+    size = "medium",
```

Note that this also increases the RAM given to this job. What about just increasing the timeout
```suggestion
    timeout = "moderate",
```